### PR TITLE
Add support for custom post type processing

### PR DESCRIPTION
**Pull request by robbinrietdijk**

### CODE REVIEW

File: php/Views/Partials/InputCheckbox.html.twig [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
-    {% if checked %}
-        checked="checked"
-    {% endif %}
-    aria-describedby="{{ name }}_hint"
->
-<p id="{{ name }}_hint">
-    {{ label }}
-</p>
+{% if id is not empty %}
+    <p>
```
Comment (jkva) [inline php/Views/Partials/InputCheckbox.html.twig]: That should be a `div`, not `p`.

File: php/PluginConfiguration.php [hunk 2]
```diff
@@ -176,9 +177,18 @@ public static function isNotProcessingUnpublishedPosts(): bool
...
 
+	/**
+	 * get optionally stored post types to process
+	 *
+	 * @return array
+	 */
     public static function getProcessedPostTypes(): array
     {
-        return ['page', 'post', 'attachment'];
+		$processedPostTypes = get_option('coyote_plugin_processed_post_types', 'not-exists');
```
Comment (jkva) [inline php/PluginConfiguration.php:188]: This is imprecision on my part, but does this imply that you can have an empty set of processed posts? That should not be a possibility. I would go as far to have `page`, `post` and `attachment` as mandatory. It makes no functional sense to use the plugin without having anything to operate on.

**Review by jkva — CHANGES_REQUESTED** [review]

File: php/Controllers/SettingsController.php [hunk 3]
```diff
@@ -335,6 +335,7 @@ private function getProfile(): ?ProfileModel {
 
     /**
      * Get registered post types in WordPress
+     * TODO: only return public post_types? Can there be post_types that aren't public but can be handeld by Coyote?
```
Comment (jkva) [inline php/Controllers/SettingsController.php]: Good Q. I would say that's for the admin to decide: if the post type is not public (yet), images within them might still require descriptions, so a batch import should still post them to the API. If they're not public post types, the user-facing side won't be affected.

Comment (robbinrietdijk) [reply php/Controllers/SettingsController.php]: updated the function to return all post_types instead of only public

**Review by jkva — APPROVED** [review]

**Pull request merged**